tfjs-backend-webgpu/src/kernels/FromPixelsImageBitmap.ts

```diff
@@ -60,7 +60,11 @@ export function fromPixelsImageBitmap(args: {
         texture: backend.fromPixelProgram.makeInputTexture(
             backend.device, imageBitmap.width, imageBitmap.height)
       },
-      {width: imageBitmap.width, height: imageBitmap.height, depth: 1});
+      {
+        width: imageBitmap.width,
+        height: imageBitmap.height,
+        depthOrArrayLayers: 1
```

Comment:
nit: It should be possible to omit `depthOrArrayLayers` as it's optional. You can also write it as an array if you want, `[imageBitmap.width, imageBitmap.height]`.

Reply:
Done.